### PR TITLE
Update stop_words.py and add name in contributors

### PR DESCRIPTION
**Pull request by jaydeepborkar**
I've added more stop words in Hindi in stop_words.py file, have linked up the resource to the same at the beginning. And as well have added my name in the contributors!

### CODE REVIEW

**Comment by svlandeg** [comment]
Hi @jaydeepborkar , thanks for this contribution! I saw you filled in the template directly, but the idea is to actually copy the template, fill it in and put the new file in the `contributors` directory, as the bot above explains :-)

**Comment by jaydeepborkar** [comment]
@svlandeg I'm extremely sorry. I just noticed. I'll make the changes and start a fresh PR if that works :)

**Comment by svlandeg** [comment]
Sure, no worries! You can keep working on this PR if you like. Just commit more changes to the same branch and they will automatically show up here.

File: .github/contributors/jaydeepborkar.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+7. Please place an “x” on one of the applicable statement below. Please do NOT
+mark both statements:
+
+    * [ ] I am signing on behalf of myself as an individual and no other person
+    or entity, including my employer, has or will have rights with respect to my
+    contributions.
+
+    * [ ] I am signing on behalf of my employer or a legal entity and I have the
+    actual authority to contractually bind that entity.
+
```
Comment (svlandeg) [inline .github/contributors/jaydeepborkar.md]: Don't forget to put an x on the appropriate place ;-)

Comment (jaydeepborkar) [reply .github/contributors/jaydeepborkar.md]: Sure, thanks :)

**Comment by jaydeepborkar** [comment]
@svlandeg thanks for all the help. I wish to work on adding support for the Hindi language and keep you updated on my progress :) Can you let me know where exactly can I keep you guys updated? Like: starting an issue with all my updates or over here itself?

**Comment by svlandeg** [comment]
You're welcome! Any help with Hindi would be very much appreciated 👍 
I think it's best to divide the work up in pieces, and whenever you have a logical unit of work done, you can open a pull request like this one and describe what you've been doing :-)

**Comment by jaydeepborkar** [comment]
@svlandeg sure, will do that :)

**Comment by ines** [comment]
Thanks a lot! 👍

**Pull request merged**